.github/ISSUE_TEMPLATE/nightly_build_failure.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Nightly build failure report
+about: Lightweight incident report for nightly build failure
+title: Nightly Build Failure XXXX-XX-XX
+labels:
+  - bug
+  - nightly-builds
+assignees: ''
```

Comment:
maybe worth auto-assigning inframundo?

Reply:
Yes 100% worth auto-assigning Inframundo. We can always reassign.